lib/tsconfig/README.md

```diff
@@ -19,7 +19,7 @@ Adapt the content of your `tsconfig.json` file as follows:
 
 ```json
 {
-	"extends": "@nationalbankbelgium/code-style/tsconfig/3.1.x",
+	"extends": "@nationalbankbelgium/code-style/tsconfig/3.2.x",
```

Comment:
Maybe it is better if you show two examples in this README: one for TS 3.1 or lower and the other for TS 3.2 or higher. What do you think?

Reply:
Let's do it 😊